data-raw/app/07_ch_flag_drug_df.R

```diff
@@ -75,10 +74,12 @@ get_geo_bnf_prop = function(index){
       BNF_PARENT = rlang::as_string(bnf),
       BNF_CHILD := {{ bnf }},
       PROP_ITEMS = janitor::round_half_up(PROP_ITEMS, 3),
-      PROP_NIC = janitor::round_half_up(PROP_NIC, 3)
+      PROP_NIC = janitor::round_half_up(PROP_NIC, 3),
+      TOTAL_ITEMS = ITEMS,
```

Comment:
I only needed to keep the already calculated totals, include them in the pivot and assign a human-friendly name to them.